generated/nixnet/nixnet.proto

```diff
@@ -918,7 +919,7 @@ message CreateSessionResponse {
 message CreateSessionByRefRequest {
   string session_name = 1;
   repeated nidevice_grpc.Session array_of_database_ref = 2;
-  string interface = 3;
```

Comment:
@asumit the issue you ran into was while trying to compile a C application right? From what I remember C client wasn't compiling because `interface` is keyword in C (which means users won't be use xnet-gRPC in C). Is that correct?

Reply:
I have added support to use sanitize_names in client files as well. Besides XNET, there was 1 API in Scope which was also using a reserved keyword "event" which got corrected as part of sanitization support.